interface/DoubleSidedGaussian.h

```diff
@@ -0,0 +1,60 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ *                                                                           *
+  * This code was autogenerated by RooClassFactory                            * 
+ *****************************************************************************/
+
+#ifndef MY_DOUBLE_Gaussian
+#define MY_DOUBLE_Gaussian
+
+#include "RooAbsPdf.h"
+#include "RooRealProxy.h"
+#include "RooCategoryProxy.h"
+#include "RooAbsReal.h"
+#include "RooAbsCategory.h"
+#include "RooGlobalFunc.h"
+
+#include "RooArgusBG.h"
+#include "RooRealVar.h"
+#include "RooDataSet.h"
+
+#include "RooGaussian.h"
+#include "RooCBShape.h"
+#include "RooExponential.h"
+#include "RooBreitWigner.h"
+
+#include "RooConstVar.h"
+#include "RooDataHist.h"
+#include "RooFitResult.h"
+#include "RooMinuit.h"
+#include "RooPlot.h"
+ 
+class DoubleSidedGaussian : public RooAbsPdf {
+public:
+  DoubleSidedGaussian() {} ; 
+  DoubleSidedGaussian(const char *name, const char *title,
+	      RooAbsReal& _x,
+	      RooAbsReal& _mean,
+	      RooAbsReal& _sig1,
+	      RooAbsReal& _sig2,
+              Double_t _yMax);
+  DoubleSidedGaussian(const DoubleSidedGaussian& other, const char* name=0) ;
+  virtual TObject* clone(const char* newname) const { return new DoubleSidedGaussian(*this,newname); }
+  inline virtual ~DoubleSidedGaussian() { }
+
+protected:
+
+  RooRealProxy x ;
+  RooRealProxy mean ;
+  RooRealProxy sig1 ;
+  RooRealProxy sig2 ;
+  Double_t evaluate() const ;
+
```

Comment:
This class doesn't implement anaytical integral capabilities, so it will be always integrated numerically. That's a major performance hit.